src/js/state/reducers/pageData.spec.js

```diff
@@ -0,0 +1,88 @@
+
+import pageDataReducer, { initState } from './pageData';
+import { SET_PAGE_DATA, REMOVE_PAGE_DATA } from '../actionTypes/pageData';
+
+
+const examplePageDataObj = [
```

Comment:
Maybe something like:
```
const pageData = {
  id: 'example_id_0',
  url: 'www.exampleurl0.com',
  fragments: []
}

const nextPageData = {
  id: 'example_id_1',
  url: 'www.exampleurl1.com',
  fragments: []
}

const allPages = [
  pageData,
  nextPageData
];
```
Then you can model expected models using references to pageData and nextPageData.